docs/cache/learn/how-it-works/read-concern.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
@allenheltondev do you have any input on where this file should live in the nav tree? I kind of think maybe it should just be a new page in the `basics` section but would love to hear your feedback.